dspace-api/src/main/java/cz/cuni/mff/ufal/dspace/discovery/SolrServiceTweaksPlugin.java

```diff
@@ -45,14 +45,15 @@ public void additionalSearchParameters(Context context,
     {
         String query = discoveryQuery.getQuery();
         // follows previous impl, but is it correct?
-        if (query == null)
-        {
+        if (query == null) {
             query = "*:*";
+        } else {
+        	query = "+(" + query + ")"
```

Comment:
you can use the description from http://stackoverflow.com/questions/27867174/how-to-tell-the-logic-like-plus-sign-with-or-in-solr including the linked article but it should be part of the code so ppl understand it